### DISCUSSION TITLE
bug: appId in capacitor config doesn't set well Bundle Identifier in XCode

### DISCUSSION THREAD

**Discussion by Aarbel** [ionic-team/capacitor#5529] [General]
As you can see, the field "Bundle identifier" is set as the app title, not the app Id.

Where does it come from ? 

<img width="458" alt="image" src="https://user-images.githubusercontent.com/25119847/159794461-17f24c25-9e09-41fe-9aa5-911cc339c948.png">
<img width="637" alt="image" src="https://user-images.githubusercontent.com/25119847/159794402-769d2b79-5250-477d-96ba-7756d80b483e.png">

**Comment by jcesarmobile** [comment]
The bundle identifier is set the first time you run `npx cap add ios` and it’s never changed again unless you remove the ios folder and run `npx cap add ios` again.
Is it possible that you set the appId to MyCapacitorApp and changed it after running add?
